### ISSUE TITLE
MacBook 8,1 2015 Cirrus 4208 support

### ISSUE THREAD

**Issue by sebl05** [leifliddy/macbook12-audio-driver#31]
Hi @leifliddy, I've been reading up on the work you've done so far making old make hardware play nice with older macs. I see in the repo that you're not supporting 8,1 2015 macbook – but you have done some extensive research into it. Couldn't you get it to work and would you still consider releasing an updated patch for it? Would be a massive help to put some new life into my old macbook that other that the speakers and bluetooth is working fine on Ubuntu 24.04. Hope to hear from you soon.

Thanks,

Seb

**Comment by sebl05** [comment]
So sorry to hear that @leifliddy. Ik hope all is well and wish you a speedy recovery.

All the best,

Seb

**Comment by 2bmoJake** [comment]
Yeah i hope you get well soon

**Comment by miguelqvd** [comment]
Hi, I hope you're fully recovered.

Any chance of getting suport for Macbook 8,1? I could help. I've put my hands on the code but I'm not familiar with programming for the firmware. I've download the technical documentation from Cirrus, though

**Comment by leifliddy** [comment]
> Hi, I hope you're fully recovered.
I didn't realize that my wife ever posted this. So I suffered a severe TBI from a motorcycle accident that occurred on my way home from work.  According to the tow truck driver, there was an oil slick on the road. Apparently I hit that oil slick whilst merging back into the right lane after overtaking a vehicle. Needless to say things didn't end up very good. In some ways I'm still dealing with aftermath of that TBI. "Normal" physical injuries are one thing, but a "brain" injury is a whole diferent beast. I mean after the accident I couldn't remember who I was....I'm going to stop here (as this isn't exactly therapy). Just know that according to the doctors I had an incredible recovery (for this type of injury). I'm still shaking the cobwebs out when it comes to certain topics like this one. Just know that I am not a kernel developer. I worked with a kernel developer (David Ulricht) to create this project. 
If I remember correctly, David Ulricht used python scripts developed by davidjo to create this kernel module. 
https://github.com/davidjo

Here's the initial bug report that I filed for this issue
https://bugzilla.kernel.org/show_bug.cgi?id=110561#c64

I'll go through my emails this weekend and will update this post with anything significant that I find. 

Thank you to everybody for your concern. It means a lot to me!

**Comment by 2bmoJake** [comment]
Thanks for this repo, it helped me get in to linux by actually being able to live with it day-day, i now have a dell xps and. I still have my little macbook and i love it and care for it. Linux is all i ever use now and i love the tinkering aspect, this repo helped me to get into it.
I’m glad to hear you have had a speedy recovery and i can only imagine that you want to get back to tinkering and i hope you do, your a helpful genius. Wish you all the best!

-Jake

**Comment by Damien0507** [comment]
Gaben here,

Just wondering if you guys have a fix yet, thanks!

**Comment by mrbubzie93** [comment]
Yes any update on this at all? It looks like someone got it working in the bugzilla bug report towards the bottom https://github.com/bluesign/macbook12-audio-driver but I wasn't able to install it via dkms or compiling...anyone had any luck?

**Comment by mrbubzie93** [comment]
Any update on this?

**Comment by mrbubzie93** [comment]
?
